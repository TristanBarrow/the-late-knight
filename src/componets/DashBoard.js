import React from 'react'
import ChessBoard from './Board';

const DashboardPage = () => (
  <div>
    <ChessBoard />
  </div>
);

export default DashboardPage;