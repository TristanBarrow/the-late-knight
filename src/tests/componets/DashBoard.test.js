import React from 'react';
import { shallow } from 'enzyme';
import DashBoard from '../../componets/DashBoard';

test('render dash board', () => {
  const wrapper = shallow(<DashBoard />);
  expect(wrapper).toMatchSnapshot();
});