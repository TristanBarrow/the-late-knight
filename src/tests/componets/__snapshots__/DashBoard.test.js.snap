// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render dash board 1`] = `
<div>
  Dashboard content
</div>
`;
